bazel-mode.el

```diff
@@ -73,6 +73,18 @@
   (when bazel-mode-buildifier-before-save
     (bazel-mode-buildifier)))
 
+(defconst bazel-mode--builtins
```

Comment:
Can you add references to the documentation here so that readers of the code don't have to search through commit messages?

Reply:
Done.